<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDO</class>
 <widget class="QWidget" name="FormDO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>325</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>10</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>10</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>571</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label1">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn1">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label9">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn9">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label17">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O17</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btn17">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label25">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O25</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="btn25">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn2">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label10">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn10">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label18">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O18</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="btn18">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label26">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O26</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="btn26">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label3">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn3">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label11">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O11</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn11">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label19">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O19</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="btn19">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label27">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O27</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="btn27">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label4">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn4">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label12">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O12</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn12">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label20">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O20</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="btn20">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label28">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O28</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="btn28">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label5">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn5">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label13">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O13</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btn13">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label21">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O21</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="btn21">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label29">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O29</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="btn29">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label6">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btn6">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label14">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O14</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btn14">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label22">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O22</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="btn22">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label30">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O30</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QPushButton" name="btn30">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label7">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O7</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btn7">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label15">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O15</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="btn15">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label23">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O23</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QPushButton" name="btn23">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLabel" name="label31">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O31</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QPushButton" name="btn31">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label8">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O8</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="btn8">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label16">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O16</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="btn16">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label24">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O24</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QPushButton" name="btn24">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="label32">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>O32</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="QPushButton" name="btn32">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Digital Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
